java-checks/src/test/java/org/sonar/java/checks/WeakSSLContextCheckTest.java

```diff
@@ -34,6 +34,15 @@ void test() {
       .verifyIssues();
   }
 
+  @Test
+  void test_java_7() {
+    CheckVerifier.newVerifier()
+      .onFile(testSourcesPath("checks/WeakSSLContextCheckJava7.java"))
```

Comment:
Instead of adding a new file with only a subset of the test, I suggest:
- Use `WeakSSLContextCheckJava8.java` for both `test_java_8`  and `test` (the behavior should be the same).
- Use the `WeakSSLContextCheck.java` for `test_java_7` (without the code changes).

Reply:
Good one!